### DISCUSSION TITLE
Get-PnPtenant is giving unauthorized

### DISCUSSION THREAD

**Discussion by Ananda-Gedela** [pnp/powershell#3304] [Q&A]
Hi Team,

We are using the Connect-PnPOnline, to connect with sharepoint using the CBA(application Id and secret). And the app has the necessary permissions and SharePoint admin role too. 
<img width="433" alt="image" src="https://github.com/pnp/powershell/assets/102895992/cfa362fa-977f-446a-879d-dfb471bc9c67">

While running the Get-PnPTenant it is giving the following error:
![image](https://github.com/pnp/powershell/assets/102895992/707dadf3-87f3-4ef6-85af-09811c75527c)
can somebody please help us why this is happening. 

Thanks

**Comment by yesh-rajawat-robomq** [comment]
Any update on this issue @Ananda-Gedela
